tests/foreman/api/test_user.py

```diff
@@ -372,3 +372,168 @@ def test_positive_update(self):
+                e.g blabla
+
+        :expectedresults: Invalid SSH key should not be added in user
+        """
+
+    @stubbed
+    @tier1
+    def test_positive_error_create_invalid_ssh_key(self):
+        """Satellite returns exception on invalid ssh key creation in user
+
+        :id: 3b2c37bd-4f95-44bc-8e57-620029535355
+
+        :steps:
+
+            1. Create new user with all the details
+            2. Attempt to add invalid string as SSH Key to above user
+                e.g blabla
+
+        :expectedresults: Satellite returns 'Fingerprint could not be
+            generated' error
+        """
+
+    @stubbed
+    @tier1
+    def test_positive_error_create_invalid_length_ssh_key(self):
+        """Satellite returns exception on invalid length of ssh key creation in
+        user
+
+        :id: 899f0c46-c7fe-4610-80f1-1add4a9cbc26
+
+        :steps:
+
+            1. Create new user with all the details
+            2. Attempt to add invalid length of SSH Key to above user
+
+        :expectedresults: Satellite returns 'Length could not be calculated'
+            error
+        """
+
+    @stubbed
+    @tier3
+    def test_positive_create_multiple_ssh_key_types(self):
+        """Multiple types of ssh keys can be added to user
+
+        :id: d1ffa908-dc86-40c8-b6f0-20650cc67046
+
+        :steps:
+            1. Create user with all the details
+            2. Add multiple types of supported ssh keys, type includes
+                rsa, dsa, ed25519, ecdsa
+
+        :expectedresults: Multiple types of supported ssh keys can be added to
+            user
+        """
+
+    @stubbed
+    @tier1
+    def test_positive_delete_ssh_key(self):
+        """Satellite Admin can delete ssh key from user
+
+        :id: 37da9052-83a7-440d-b24c-9d4458f011e3
+
+        :steps:
+
+            1. Create new user with all the details
+            2. Add SSH Key to above user
+            3. Delete the ssh-key from user
+
+        :expectedresults: SSH key should be deleted from user
+        """
+
+    @stubbed
+    @tier3
+    def test_positive_ssh_key_in_host_enc(self):
+        """SSH key appears in host ENC output
+
+        :id: 4b70a950-e777-4b2d-a83d-29279715fe6d
+
+        :steps:
+
+            1. Create user with all the details
+            2. Add ssh key in above user
+            3. Provision a new host from the above user
+            4. Check new hosts ENC output
+
+        :expectedresults: SSH key should be added to host ENC output
+        """
+
+    @stubbed
+    @tier2
+    def test_positive_list_users_ssh_key(self):
+        """Satellite lists users ssh keys
+
+        :id: 8098e74a-d81e-4410-b744-435901bd70c0
+
+        :steps:
+
+            1. Create user with all the details
+            2. Add SSH key in above user
+            3. List all the ssh keys of above user
+
+        :expectedresults: Satellite should list all the SSH keys of user
+        """
+
+    @stubbed
+    @tier2
```

Comment:
Not sure that it is tier2, looks same for me as other tier1 tests.
Also `:CaseLevel: System` is missing for tier2.